Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -276,7 +297,7 @@ public function removeUnusedCommand(string $assetSource = '', bool $quiet = fals
 
         if ($assumeYes === false) {
             if (!$this->output->askConfirmation(sprintf('Do you want to remove <b>%s</b> unused assets?', $unusedAssetCount))) {
-                exit(1);
+                exit;
```

Comment:
```suggestion
                exit(1);
```

I think this should stay, it is an unsuccessful run if nothing was deleted after all?

Reply:
why not `$this->quit(1);`?

Reply:
or that, sure!